rollup.config.mjs

```diff
@@ -0,0 +1,24 @@
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import serve from 'rollup-plugin-serve';
+
+const production = !process.env.ROLLUP_WATCH,
+
+ modernConfig = {
+  input: {
+    ['auro-toast__bundled']: './index.js',
+  },
+  output: {
+    format: 'esm',
+    dir: 'dist/'
+  },
+  plugins: [
+    nodeResolve(),
+    !production &&
+      serve({
+        open: true,
+        openPage: '/docs/'
+      })
+  ]
+};
+
+export default [modernConfig];
```

Comment:
Need to update rollup.config: https://github.com/AlaskaAirlines/auro-combobox/blob/main/rollup.config.mjs

Reply:
@jordanjones243 again, please follow up with @cynthiaricomendoza-alaskaair on this. 

Also, this model is not part of the generator. I was unaware that this was a pattern that we are actively doing with all components. 

@Patrick-Daly-AA there should be an issue created with the WC-Generator to add this functionality. Please work with @jordanjones243 to define that specification.

Reply:
met with Jordan yesterday and pushed up changes for this!